packages/vitest-pool-workers/package.json

```
@@ -70,9 +70,9 @@
 		"zod": "^3.20.6"
 	},
 	"peerDependencies": {
-		"@vitest/runner": "1.3.0",
-		"@vitest/snapshot": "1.3.0",
-		"vitest": "1.3.0"
+		"@vitest/runner": "1.5.0",
+		"@vitest/snapshot": "1.5.0",
+		"vitest": "1.5.0"
```

Should we make these peer dependencies a bit more lenient?
I feel like bumping these strict dependencies is a breaking change to the library, forcing developers to update to 1.5.0 even if they would prefer to stay on 1.3.0 or 1.4.0.

The reason why this was not originally set to `^1.3.0` was that we are using some internal APIs that could have a breaking change within a minor release.
But that being said, the Vitest team are aware of what APIs we are using and are onboard with these becoming public and stabilized.

We should fast-track that stabilization but in the meantime, should we be more lenient here?

Perhaps we could go with:

```suggestion
		"@vitest/runner": "1.3.x||1.4.x||1.5.x",
		"@vitest/snapshot": "1.3.x||1.4.x||1.5.x",
		"vitest": "1.3.x||1.4.x||1.5.x"
```

---

Thanks Pete! I'm very happy to do this, and tweak the specific version enforcer to just yell loudly instead of exiting entirely. Something like `>=1.3.0 || <= 1.5.x` might make this version a little easier to maintain in future as 1.6.0 gets released, etc.

---

Done!